cmd/dcrctl/httpclient.go

```diff
@@ -72,7 +72,7 @@ func newHTTPClient(cfg *config) (*http.Client, error) {
 func sendPostRequest(marshalledJSON []byte, cfg *config) ([]byte, error) {
 	// Generate a request to the configured RPC server.
 	protocol := "http"
-	if !cfg.NoTLS {
+	if !cfg.NoTLS && !cfg.TLSSkipVerify {
```

Comment:
This doesn't seem correct.   Skipping verify shouldn't not disable https;  it should disable cert verification.

Reply:
https://golang.org/pkg/crypto/tls/#Config

InsecureSkipVerify